Add very quick start (#20)
premAI-io/prem-operator: README.md

### Overview

> Deploy AI models and apps to Kubernetes without hitting every pitfall.

Reviewer: ```suggestion
Deploy AI models and apps to Kubernetes without hitting any pitfalls and feeling overwhelmed by the ginormous YAMLs.
```

Author: I don't think we can claim they won't hit any pitfalls :-)

### Quick Start

Deploy the operator and chat with Hermes in 2 or 3 steps. Note that you need a NVIDIA GPU with 8GB of RAM.

> 1. Install the NVIDIA Operator if not present already

Reviewer: ```suggestion
1. Install the NVIDIA Operator if not present already (not needed if the host already has the relevant drivers installed, and is manually managed)
```

Reviewer: I am not sure if we should add all this information so a more succint version of this.

Author: The operator is at least needed for node labeling unless they label the nodes themselves. It is possible to install the operator without it configuring the runtime and drivers.

I don't really want to get into whether it's possible to use an NVIDIA card without the operator. Especially not in the quickstart example(s).

2. Install the Prem Operator
    ```
    $ helm install latest oci://registry-1.docker.io/premai/prem-operator-chart
    ```

3. Deploy the Hermes + Big-AGI example and forward the ports
    ```
    $ kubectl -f exmaples/big-agi.yaml
    $ kubectl port --address localhost --port 8080 forward big-agi-service --port 3000
    ```

> If you browse to localhost:3000 you'll be able to beging chatting as soon as

Reviewer: ```suggestion
If you browse to localhost:3000 you'll be able to begin chatting as soon as
```

> LocalAI downloads the model.

Reviewer: ```suggestion
LocalAI downloads the model. All queries after the first one, where the model gets downloaded should be much faster.
```

Author: This is clear in the video (which I haven't converted to a GIF yet, but you can see here https://youtu.be/MN8WpMG9PKs)

I think the information here has to be really minimal.
